src/libstore/build/derivation-goal.cc

```diff
@@ -868,11 +868,7 @@ void runPostBuildHook(
 }
 
 
-void appendLogTailErrorMsg(
```

Comment:
Is this really a good change?

Reply:
See the commit message --- the other parameters I think don't belong in the interface but rather in the "closure environment" of the implementation.

Really longer term, this UI-focused thing is a layer violation and should not be in here at all.